extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitterModule.java

```diff
@@ -68,6 +68,18 @@ public Emitter apply(String s)
           }
         }
     );
-    return new GraphiteEmitter(graphiteEmitterConfig, emitters);
+
+    List<Emitter> requestLogEmitters = Lists.transform(
+        graphiteEmitterConfig.getRequestLogEmitters(),
+        new Function<String, Emitter>()
+        {
+          @Override
+          public Emitter apply(String s)
+          {
+            return injector.getInstance(Key.get(Emitter.class, Names.named(s)));
```

Comment:
This has the potential to fail very late. Suggest eagerly creating the list with `ImmutableList.copyOf`